script.js

```diff
@@ -1,14 +1,222 @@
+    name: 'Proffesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-upsrequired. has been the industry s standard dummy text ever since the 1500s, when an unknown printer tooka standard dummy text.',
+    featuredImage: 'assets/images/Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'bootstrap', 'React', 'Ruby', 'JavaScript'],
+    projecDetails: 'see-project',
+    linkToLive_version: 'See Live',
+    linkToSource: 'See Source',
+  },
+  {
+    id: 5,
+    name: 'Proffesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-upsrequired. has been the industry s standard dummy text ever since the 1500s, when an unknown printer tooka standard dummy text.',
+    featuredImage: 'assets/images/Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'bootstrap', 'React', 'Ruby', 'JavaScript'],
+    projecDetails: 'see-project',
+    linkToLive_version: 'See Live',
+    linkToSource: 'See Source',
+  },
+  {
+    id: 6,
+    name: 'Proffesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-upsrequired. has been the industry s standard dummy text ever since the 1500s, when an unknown printer tooka standard dummy text.',
+    featuredImage: 'assets/images/Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'bootstrap', 'React', 'Ruby', 'JavaScript'],
+    projecDetails: 'see-project',
+    linkToLive_version: 'See Live',
+    linkToSource: 'See Source',
+  },
+  {
+    id: 7,
+    name: 'Proffesional Art Printing Data',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-upsrequired. has been the industry s standard dummy text ever since the 1500s, when an unknown printer tooka standard dummy text.',
+    featuredImage: 'assets/images/Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'bootstrap', 'React', 'Ruby', 'JavaScript'],
+    projecDetails: 'see-project',
+    linkToLive_version: 'See Live',
+    linkToSource: 'See Source',
+  },
+];
+
+const firstCard = document.querySelector('.recent-works');
+recent.map((element) => {
+  firstCard.innerHTML = `
+   <img src="${element.featuredImage}" alt="Multi post story" class="work-img" />
+   <div class="recent-work-second-block">
+     <h3 class="works-title blue-1">${element.name}</h3>
+     <p class="font-montesori blue-2">
+     ${element.description}
+     </p>
+     <ul class="tools-container">
+       <li class="tools">${element.technologies[0]}</li>
+       <li class="tools">${element.technologies[1]}</li>
+       <li class="tools">${element.technologies[2]}</li>
+       <li class="tools">${element.technologies[3]}</li>
+
+   
+     </ul>
+     <button class="see-project-btn" type="button">${element.projecDetails}</button>
+   </div>`;
+  return firstCard;
+});
+
+const cards = document.querySelector('.pjt');
```

Comment:
Great job @aubinleyoung. As the previous reviewer requested, kindly align the heading as seen on the Figma template.

Figma template
<img width="263" alt="Screenshot 2022-12-23 at 8 27 16 PM" src="https://user-images.githubusercontent.com/25887253/209382280-8dd5d629-e092-48e4-a45f-21dfd37bde4e.png">

Your design:
<img width="390" alt="Screenshot 2022-12-23 at 8 26 15 PM" src="https://user-images.githubusercontent.com/25887253/209382414-cbde3481-c377-4093-b58b-6ed139add97d.png">